add_source_file

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
```

Comment:
On non-Windows, that's python2, not python3. Copy & paste the stuff from another of our scripts:
```
#!/usr/bin/env python3
# encoding: utf-8
```

Reply:
Hmm i thought Debian was the last remaining holdout there. Updated